### PR TITLE
Migration to GitHub Actions

### LINKED ISSUE #68: Migration to GitHub Actions
As suggested here:
https://github.com/jhipster/generator-jhipster-blueprint/issues/66#issuecomment-598056421

### PR DESCRIPTION
**Pull request by murdos**
Fixes #68

### CODE REVIEW

File: .github/workflows/github-ci.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                    mkdir -p $APP_FOLDER/.jhipster/ && cd $APP_FOLDER
+                    cp $JHIPSTER_SAMPLES/ngx-blueprint/.yo-rc.json $APP_FOLDER
+                    curl https://raw.githubusercontent.com/jhipster/generator-jhipster/master/test-integration/samples/.jhipster/BankAccount.json -o .jhipster/BankAccount.json
+                    curl https://raw.githubusercontent.com/jhipster/generator-jhipster/master/test-integration/samples/.jhipster/Label.json -o .jhipster/Label.json
+                    curl https://raw.githubusercontent.com/jhipster/generator-jhipster/master/test-integration/samples/.jhipster/Operation.json -o .jhipster/Operation.json
+
+                    jhipster --blueprints helloworld --force --no-insight --with-entities
+                    [ -f dummy.txt ]
+
+                    # generate servies
```
Comment (pascalgrimaud) [inline .github/workflows/github-ci.yml]: typo: services

File: .github/workflows/github-ci.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+on: [push, pull_request]
+jobs:
+    unit-tests:
+        name: unit tests
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v2
+            -   uses: actions/setup-node@v1
+                with:
+                    node-version: 12.14.0
```
Comment (pascalgrimaud) [inline .github/workflows/github-ci.yml]: can be upgrade to 12.16.1 to be consistent with main generator

File: .github/workflows/github-ci.yml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            -   name: Install node.js packages
+                run: npm install
+            -   name: Run generator tests
+                run: npm test
+    integration-tests:
+        name: integration tests
+        runs-on: ubuntu-latest
+        timeout-minutes: 10
```
Comment (pascalgrimaud) [inline .github/workflows/github-ci.yml]: are you sure it's enough ?
I would increase to 20 to be sure lol :-D

**Comment by pascalgrimaud** [comment]
LGTM, only minor comments

**Comment by murdos** [comment]
Thanks for the review @pascalgrimaud !

**Pull request merged**